core/store/orm/orm.go

```diff
@@ -201,7 +196,7 @@ func (orm *ORM) FindInitiator(ID int64) (initr models.Initiator, err error) {
 		return initr, err
 	}
 	return initr, orm.DB.
-		Set("gorm:auto_preload", true).
+		Preload(clause.Associations).
```

Comment:
I hate this with all my heart, can we make it explicit please?

Reply:
hmm this feels pretty explicit to me - its reads like "preload all the associations"?